src/method_ilan.jl

```diff
@@ -76,6 +76,8 @@ function ilan(
     check_error_every=30,
     inner_solver_method=DefaultInnerSolver(),
     Compute_Bmul_method::Type{T_y0}=Compute_Bmul_method_Auto,
+    proj_solve=true,
+    inner_logger=0
     )where{T<:Number,T_orth<:IterativeSolvers.OrthogonalizationMethod,T_y0<:Compute_Bmul_method}
 
     @parse_logger_param!(logger)
```

Comment:
`@parse_logger_param!` should be called also for the inner logger, cf iar and other methods with inner solvers.